biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs

```diff
@@ -12,7 +12,8 @@
                     will return aggregate counts per study and consent. From PIC-SURE Open Access you have the ability
                     to request access to studies you are currently not authorized to access. Visit the
                     <a href="https://tinyurl.com/PIC-SURE-User-Guide"
```

Comment:
```suggestion
                    <a href="https://tinyurl.com/BDC-PIC-SURE-User-Guide"
```

Reply:
I don't think this template is used anywhere. The router module includes it as a dependency, but doesn't use it anywhere.

Reply:
Lets delete it then